### PR TITLE
Improve tests

### PR DESCRIPTION
**Pull request by brendt**
This PR adds tests for all commands.

### CODE REVIEW

**Comment by freekmurze** [comment]
My gut says me these tests are too low level are more testing the implementation then the functionality itself. 

I'm thinking tests the just test the response given at the http layer is enough.

Pinging @sebastiandedeyne for second opinion.

File: tests/TestCase.php [hunk 1]
```diff
@@ -11,8 +11,18 @@
...
+
+        return $this->post($url, compact('command'));
+    }
+
+    protected function getFlashMessage(): ?string
+    {
+        $flash = app('session.store')
+            ->get('flash_notification');
+
+        return $flash ? $flash->first()->message : null;
```
Comment (sebastiandedeyne) [inline tests/TestCase.php]: If you always return a string you can drop the nullable type hint. I prefer this when null and an empty value of the type have the same meaning in the context.

**Comment by sebastiandedeyne** [comment]
> I'm thinking testing the response for a given command at the http layer is enough and gives us enough confidence that everything is working properly.

Isn't that what's happening here?

**Comment by freekmurze** [comment]
Indeed, these tests are actually performed via the http layer (glanced a bit too quickly over them on my iphone screen while in transit). 

Ok for me too!

File: tests/TestCase.php (hunk 1 above)
Comment (brendt) [reply tests/TestCase.php]: To me that's a different meaning: `null` means no flash message is set, while an empty string could also mean an empty flash message was set.

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**